src/components/SavedRecipes.tsx

```diff
@@ -21,6 +26,34 @@ const SavedRecipes = () => {
       });
   }, []);
 
+  const handleRecipeSearch = useCallback((params: string) => {
+    const searchedRecipes = recipes.filter((recipe: SavedRecipe) => {
+        const search = params.toLowerCase();
```

Comment:
If would be more efficient if you did this parsing on line 16 (then you only do it once instead of multiple times).

Reply:
See comment above w/ example

Reply:
<img width="529" alt="Screen Shot 2023-08-11 at 09 25 03" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/36e01633-96ea-4092-960c-7436f4269198">
<img width="507" alt="Screen Shot 2023-08-11 at 09 25 10" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/5927bf4b-7a35-42bf-ba58-16bf87488d5e">
Is that what you mean? Then there is a type issue...

Reply:
Maybe this is because we don't necessary have params...